### PR TITLE
不要なimportを削除

### PR DESCRIPTION
**Pull request by matsuuramasakazu**
## Summary by CodeRabbit

* **Chores**
  * Added GitHub automation for automated reviews (configured for Japanese, branch/path/draft/documentation filters)
  * Removed unused imports and small cleanup across the codebase
* **Bug Fixes**
  * More robust image loading and error handling to avoid silent failures and race conditions
  * Improved markup parsing, hashing and geometry edge-case handling for more consistent rendering
* **Tests**
  * Added validation tests and file-not-found tests; updated existing image-utils test to mock existence checks

### CODE REVIEW

File: .github/coderabbit.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    - "dist/**"
+    - "build/**"
+    - "**/*.min.js"
+
+  # ドキュメントの変更は無視
+  ignore_documentation_changes: true
+
+# 自動レビューの対象ブランチ（例: main）
+# auto_review:
+  base_branches: ["main"]
```
[the remark this answers is not in the document]
Comment (matsuuramasakazu) [reply .github/coderabbit.yaml]: ご指摘ありがとうございます。`.github/coderabbit.yaml` の構造をドキュメント通りに整理し、`auto_review` セクションのネストと `path_filters` の除外記法（`!`）を修正しました。最新のコミット (`839618d`) で反映済みです。

File: py_mind_memo/image_utils.py [hunk 2]
```diff
@@ -1,23 +1,30 @@
...
     """指定されたサイズに収まるように最小の整数サンプリングレートを計算する"""
     if width <= max_width and height <= max_height:
         return 1
     
-    # 幅と高さの比率を計算（切り上げ相当の整数計算）
-    sample_x = (width + max_width - 1) // max_width
-    sample_y = (height + max_height - 1) // max_height
+    # 幅と高さの比率を計算（より明確に math.ceil を使用）
+    sample_x = math.ceil(width / max_width)
+    sample_y = math.ceil(height / max_height)
```
[the remark this answers is not in the document]
Comment (matsuuramasakazu) [reply py_mind_memo/image_utils.py]: `max_width` / `max_height` が 0 以下の場合に `ValueError` を投げるようにバリデーションを追加しました。併せてテストケースも追加し、正常に動作することを確認済みです。

File: tests/test_image_utils.py [hunk 3]
```diff
@@ -16,10 +16,20 @@ def test_calculate_subsample(self):
...
+            calculate_subsample(100, 100, 100, -5)
 
-    def test_file_to_base64(self):
+    def test_file_to_base64_not_found(self):
+        # ファイルが存在しない場合、FileNotFoundError が発生することを確認
+        with self.assertRaises(FileNotFoundError):
+            file_to_base64("non_existent.png")
+
+    @patch("os.path.exists", return_value=True)
+    def test_file_to_base64(self, mock_exists):
```
[the remark this answers is not in the document]
Comment (matsuuramasakazu) [reply tests/test_image_utils.py]: ご指摘の通り、`file_to_base64` から `os.path.exists` の呼び出しを削除したため、テスト側のパッチと引数が不要になっていました。これらを削除してテストコードをクリーンアップしました。ありがとうございます。
